ssg/build_cpe.py

```diff
@@ -84,19 +83,17 @@ def add_cpe_item(self, cpe_item):
     def _is_name(self, ref):
         return ref.startswith("cpe:")
 
-    def _is_parametrized(self, ref):
-        return re.search(r'^\w+\[\w+\]$', ref)
-
-    def get_cpe(self, ref):
+    def get_cpe(self, cpe_id_or_name):
         try:
-            if self._is_name(ref):
-                return self.cpes_by_name[ref]
+            if self._is_name(cpe_id_or_name):
```

Comment:
@matejak Should the `_is_name` be moved to Symbol class as well?

Reply:
Good catch, the class `ProductCPEs` definitely isn't the one that has the knowledge about how data structures format stuff themselves.
The naming `is_name` is also confusing - for example, to me "Jan" is a name, whereas `human:Jan` is not what I would identify as name, it's perhaps some sort of an ID.